runtime/tests/interpreter/array_test.go

```diff
@@ -1,87 +0,0 @@
-/*
```

Comment:
Refactored to https://github.com/onflow/cadence/pull/2595/files#diff-c3306e7eb27241cc245335501f5b9cf9dcd45474613bc9e0c0926f7a71666973